examples/tp/hello_tm.py

```diff
@@ -6,7 +6,7 @@
 # following terms and conditions apply:
 #
 # This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero Public License version 3 as
+# it under the terms of the GNU General Public License version 3 as
```

Comment:
@rcrowder 
@rhyolight 
Sorry. My bad. That was not intentional. I worked on old code with old license header in the initial commit (didn't realize untill it was pointed out).  While trying to make recommended changes, I changed only one reference of license. I did not observe other line of it. I will be careful from now on.